crates/lib3h/src/time.rs

```diff
@@ -1,25 +1,36 @@
 use std::time::{SystemTime, UNIX_EPOCH};
 
+pub static mut START_TIME: SystemTime = UNIX_EPOCH;
+
 pub fn since_epoch_ms() -> u64 {
-    let start = SystemTime::now();
-    let since_the_epoch = start
-        .duration_since(UNIX_EPOCH)
-        .expect("Time went backwards");
+    let since_the_epoch;
+    unsafe {
```

Comment:
Do you happen to know why we need this in an `unsafe` block ?

Reply:
Because using a global variable is unsafe and START_TIME is one.

Reply:
Oh you mean a mutable global static one, this is indeed really unsafe ^^'

Reply:
Humm I'm not sure about what you are trying to do then ^^' 

Could you please tell me more about what this ```rust pub fn since_epoch_ms() ``` function is supposed to do ?